cargo install panamax has a compilation error

As stated fails to install panamax.
System: Manjaro 5.13
KDE: 5.24
Hardware: AMD Ryzen 7 2700


Error message:
```bash
   cargo install panamax
   ...
   Compiling panamax v1.0.4
error[E0599]: no method named `progress_chars` found for enum `Result` in the current scope
   --> /home/verontrix/.cargo/registry/src/github.com-1ecc6299db9ec823/panamax-1.0.4/src/crates.rs:127:18
    |
127 |                 .progress_chars("  ")
    |                  ^^^^^^^^^^^^^^ method not found in `Result<ProgressStyle, indicatif::style::TemplateError>`

error[E0308]: mismatched types
   --> /home/verontrix/.cargo/registry/src/github.com-1ecc6299db9ec823/panamax-1.0.4/src/crates.rs:131:27
    |
131 |     pb.enable_steady_tick(10);
    |                           ^^ expected struct `Duration`, found integer

error[E0599]: no method named `progress_chars` found for enum `Result` in the current scope
   --> /home/verontrix/.cargo/registry/src/github.com-1ecc6299db9ec823/panamax-1.0.4/src/crates.rs:190:18
    |
190 |                 .progress_chars("█▉▊▋▌▍▎▏  ")
    |                  ^^^^^^^^^^^^^^ method not found in `Result<ProgressStyle, indicatif::style::TemplateError>`

error[E0308]: mismatched types
   --> /home/verontrix/.cargo/registry/src/github.com-1ecc6299db9ec823/panamax-1.0.4/src/crates.rs:194:27
    |
194 |     pb.enable_steady_tick(10);
    |                           ^^ expected struct `Duration`, found integer

error[E0599]: no method named `set_draw_rate` found for struct `ProgressBar` in the current scope
   --> /home/verontrix/.cargo/registry/src/github.com-1ecc6299db9ec823/panamax-1.0.4/src/crates.rs:195:8
    |
195 |     pb.set_draw_rate(10);
    |        ^^^^^^^^^^^^^ help: there is an associated function with a similar name: `set_draw_target`

error[E0308]: mismatched types
   --> /home/verontrix/.cargo/registry/src/github.com-1ecc6299db9ec823/panamax-1.0.4/src/rustup.rs:343:27
    |
343 |     pb.enable_steady_tick(10);
    |                           ^^ expected struct `Duration`, found integer

error[E0308]: mismatched types
   --> /home/verontrix/.cargo/registry/src/github.com-1ecc6299db9ec823/panamax-1.0.4/src/rustup.rs:663:27
    |
663 |     pb.enable_steady_tick(10);
    |                           ^^ expected struct `Duration`, found integer

error[E0599]: no method named `progress_chars` found for enum `Result` in the current scope
  --> /home/verontrix/.cargo/registry/src/github.com-1ecc6299db9ec823/panamax-1.0.4/src/crates_index.rs:45:18
   |
45 |                 .progress_chars("  ")
   |                  ^^^^^^^^^^^^^^ method not found in `Result<ProgressStyle, indicatif::style::TemplateError>`

error[E0308]: mismatched types
  --> /home/verontrix/.cargo/registry/src/github.com-1ecc6299db9ec823/panamax-1.0.4/src/crates_index.rs:51:27
   |
51 |     pb.enable_steady_tick(10);
   |                           ^^ expected struct `Duration`, found integer

error[E0599]: no method named `progress_chars` found for enum `Result` in the current scope
   --> /home/verontrix/.cargo/registry/src/github.com-1ecc6299db9ec823/panamax-1.0.4/src/rustup.rs:264:18
    |
264 |                 .progress_chars("█▉▊▋▌▍▎▏  ")
    |                  ^^^^^^^^^^^^^^ method not found in `Result<ProgressStyle, indicatif::style::TemplateError>`

Some errors have detailed explanations: E0308, E0599.
For more information about an error, try `rustc --explain E0308`.
error: failed to compile `panamax v1.0.4`, intermediate artifacts can be found at `/tmp/cargo-installOtGonN`

Caused by:
  could not compile `panamax` due to 10 previous errors
```

Cloning panamax successfully builds.  However, trying to install the build using `cargo install --path` fails . .  Same error message.

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,8 +11,12 @@ RUN cargo build --release $CARGO_BUILD_EXTRA
 FROM debian:latest
 
 COPY --from=builder /app/target/release/panamax /usr/local/bin
-RUN apt update
-RUN apt install -y libssl1.1 ca-certificates
+
+RUN apt update \
+  && apt install -y \
+    ca-certificates \
+    git \
+    libssl1.1
 
 ENTRYPOINT [ "/usr/local/bin/panamax" ]
 CMD ["--help"]

--- a/README.md
+++ b/README.md
@@ -24,11 +24,11 @@ Alternatively, you can clone this repository and `cargo build` or `cargo run` wi
 Panamax is available as a docker image, so you can run:
 
 ```
-$ docker run -it -v /path/to/mirror/:/mirror panamaxrs/panamax init /mirror
+$ docker run --rm -it -v /path/to/mirror/:/mirror panamaxrs/panamax init /mirror
 (Modify /path/to/mirror/mirror.toml as needed)
-$ docker run -it -v /path/to/mirror/:/mirror panamaxrs/panamax sync /mirror
+$ docker run --rm -it -v /path/to/mirror/:/mirror panamaxrs/panamax sync /mirror
 (Once synced, serve the mirror)
-$ docker run -it -v /path/to/mirror/:/mirror -p8080:8080 panamaxrs/panamax serve /mirror
+$ docker run --rm -it -v /path/to/mirror/:/mirror -p8080:8080 panamaxrs/panamax serve /mirror
 ```
 
 Alternatively, you can run panamax in a bare-metal environment like below.

--- a/README.md
+++ b/README.md
@@ -12,7 +12,7 @@ Panamax is a tool to mirror the Rust and crates.io repositories, for offline usa
 Panamax is itself available on crates.io, and can be installed via:
 
 ```
-$ cargo install panamax
+$ cargo install --locked panamax
 ```
 
 Alternatively, you can clone this repository and `cargo build` or `cargo run` within it.

